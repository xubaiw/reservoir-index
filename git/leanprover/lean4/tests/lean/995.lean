example : ∀(x : Nat){h : x = x}, Nat := by
  intro x
  match x with
  | 0 => _
  | n + 1 => _

example (x : Nat) : ∀{h : x = x}, Nat := by
  match x with
  | 0 => _
  | n + 1 => _
