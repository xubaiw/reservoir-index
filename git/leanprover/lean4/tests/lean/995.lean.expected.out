995.lean:1:40-5:14: error: unsolved goals
case match_1
x : Nat
⊢ {h : 0 = 0} → Nat

case match_2
x n : Nat
⊢ {h : n + 1 = n + 1} → Nat
995.lean:7:41-10:14: error: unsolved goals
case match_1
x : Nat
⊢ {h : 0 = 0} → Nat

case match_2
x n : Nat
⊢ {h : n + 1 = n + 1} → Nat
